common/persistence/visibility/store/elasticsearch/client/client.go

```diff
@@ -68,10 +72,7 @@ type (
 	}
 
 	IntegrationTestsClient interface {
```

Comment:
You can add `Client` to `IntegrationTestsClient` so you won't need to create two clients.